packages/core-data/src/hooks/use-entity-record.ts

```diff
@@ -92,6 +92,8 @@ export default function useEntityRecord< RecordType >(
 	};
 }
 
+export default useEntityRecord;
```

Comment:
Without a separate named export, docgen did not generate the README.md properly

Reply:
@dmsnell contributed a few fixes to docgen recently to make it work better with TypeScript. It's probably another place where we need to tweak it.

Reply:
Your patch seems to have done the trick, so I think it's not as much about changing docgen as it is about clarifying how to use it.

Reply:
it might be the import that does `default as useSomething` 👍🏻